pkgs/development/compilers/julia/README.md

```diff
@@ -0,0 +1,24 @@
+Julia
```

Comment:
Please move this file into docs/language-frameworks.

Reply:
This README seems to be package-developer-only and as far as I see doc/language-frameworks is supposed to be at least of some use to users.

Reply:
@7c6f434c is correct, there is also precedence for this when you look at for example `elm`, `libreoffice`, etc.